src/robusta/core/sinks/robusta/rrm/prometheus_alert_resource_manager.py

```diff
@@ -0,0 +1,288 @@
+                self.set_account_resource_status(status_type=AccountResourceStatusType.error, info=None)
+
+            logging.error(f"Error occured while making prometheus alerts: {e}", exc_info=True)
+
+            raise e
+
+    def start(self, active_alert_rules: List[PrometheusAlertRule]) -> int:
+        synced_item_count = 0
+        try:
+            sorted_active_rules: List[PrometheusAlertRule] = sorted(active_alert_rules, key=lambda x: x.alert)
+
+            # Each CRD file has a limit of 700 rules. This limit is defined by the
+            # [MAX_ALLOWED_RULES_PER_CRD_ALERT] variable. To adhere to this limit,
+            # we calculate the number of iterations required. This calculation is
+            # based on dividing the total number of active rules by the maximum
+            # rules permitted per CRD.
+
+            if self.__max_allowed_rules_per_crd == 0:
+                max_iterations = 0
+            else:
+                max_iterations = math.ceil(len(sorted_active_rules) / self.__max_allowed_rules_per_crd)
+
+            for next_iteration in range(0, max_iterations):
+                start_index = next_iteration * self.__max_allowed_rules_per_crd
+                end_index = (next_iteration + 1) * self.__max_allowed_rules_per_crd
+                name = f"{self.__crd_name}--{(next_iteration + 1)}"
+
+                sliced_alerts = sorted_active_rules[start_index:end_index]
+                _, item_length = self.__create_crd_file(name=name, alerts=sliced_alerts)
+                synced_item_count += item_length
+
+            # When a user disables or deletes alerts from the sass platform, they may be
+            # removed from [active_alert_rules] and consequently from the cluster
+            # In certain edge cases, there's a possibility that these rules aren't entirely
+            # cleared from the cluster or cache.
+            # This typically happens when the [max_iterations] value is smaller than the size of the cached dictionary.
+            # As a result, the above loop might not cover all crd file buckets, leaving
+            # behind residual rules in both the cluster and cache.
+            # The loop below ensures these residues are effectively cleaned up.
+            cached_length = len(self.__crd_files_cache.keys())
+            for next_iteration in range(max_iterations + 1, cached_length + 1):
+                name = f"{self.__crd_name}--{next_iteration}"
+                self.__delete_crd_file(name=name)
+
+            return synced_item_count
+        except Exception as e:
+            logging.error(f"Error occured while creating prometheus alerts: {e}", exc_info=True)
+
+            raise e
+
+    def __get_snapshot_body(self, name: str, rules: List[dict]):
+        return {
+            "apiVersion": self.__k8_apiVersion,
+            "kind": self.__kind,
+            "metadata": {
+                "name": name,
+                "labels": {
+                    "release": "robusta",
```

Comment:
`release` might not be `robusta`
Please take it from the env var named RELEASE_NAME